build.gradle

```diff
@@ -55,8 +55,8 @@ ext {
     androidxTestCoreVersion = '1.4.0'
     assertJVersion = '3.11.1'
     espressoVersion = '3.4.0'
-    mockitoCoreVersion = "3.3.3"
-    nhaarmanMockitoVersion = "2.2.0"
+    mockitoVersion = "4.8.1"
+    mockitoKotlinVersion = '4.0.0'
```

Comment:
❓ Shouldn't we leave those versions to be declared by the version catalog?

Reply:
This is a nice idea @wzieba , but `version catalog` is only configured for the `FluxC` for the time being. Neither WPAndroid, nor WCAndroid are utilizing `version catalog` atm. Is that what you meant? 🤔

As such, for now, I wouldn't involve doing anything version catalog related.

Reply:
Oh sorry! I missed that "tiny" detail. Thanks 😅